BIDECDSA.go

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2018, 1Kosmos Inc. All rights reserved.
+ * Licensed under 1Kosmos Open Source Public License version 1.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of this license at
+ *    https://github.com/1Kosmos/1Kosmos_License/blob/main/LICENSE.txt
+ */
+
+package gohelperfiles
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	base64 "encoding/base64"
+	"fmt"
+	"io"
+	"log"
+
+	secp256k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
+)
+
+const (
+	NoPadding Padding = iota
+)
+
+const (
+	GCM CipherMode = iota
+)
+
+type CipherMode int
+type Padding int
+
+const ENCRYPT = "encrypt"
+const DECRYPT = "decrypt"
+
+type AES struct {
+	CipherMode CipherMode
+	Padding    Padding
+}
+
+func EcdsaHelper(method, text string, key []byte) (string, error) {
+	if method == ENCRYPT {
+		return Encrypt(text, key)
+	} else if method == DECRYPT {
+		return Decrypt(text, key)
+	} else {
+		return "", fmt.Errorf("EcdsaHelper invalid method %s", method)
+	}
+}
+
+func Encrypt(plainText string, key []byte) (string, error) {
+	aes, err := aes.NewCipher(key)
```

Comment:
based on below "gcm" function, is this one doing CBC? We must only use GCM please

Reply:
no Galois Counter Mode (GCM)